flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -3228,5 +3228,8 @@
     },
     "net.rpcs3.RPCS3": {
         "module-rpcs3-source-git-branch": "Needed for custom updater to update once a day per upstream recommendation."
+    },
+    "com.teamspeak.TeamSpeak": {
+    	"finish-args-unnecessary-xdg-config-access": "TeamSpeak 5 plugins installation directory."
```

Comment:
There is an entry in https://github.com/flathub-infra/flatpak-builder-lint/blob/7ba6e745c3d14f3358a0a48d34b493d0db08a675/flatpak_builder_lint/staticfiles/exceptions.json#L1556

add the new exception there.

Reply:
I wasn't aware that there was already an entry for TeamSpeak, let me update it.